JavaScript/unit_testing/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "unit_testing",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
and the test command in this package json file again this "test": "updated here"